sdk/identity/identity/CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## 1.2.3 (Unreleased)
 
+- Fixed Azure Stack support for the NodeJS version of the `InteractiveBrowserCredential`.
```

Comment:
Can we add a link to the issue that was fixed?

Reply:
I will!